markets/perps-market/contracts/interfaces/IMarketEvents.sol

```diff
@@ -0,0 +1,13 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.11 <0.9.0;
+
+interface IMarketEvents {
```

Comment:
any others around that we can throw in here maybe? 👀

Reply:
Most modules will have their events defined in their own interface.  I had a look, didn't see any clear ones to moce